### PR TITLE
Patch based 2.7

### PR DESCRIPTION
**Pull request by franciscocorrales**
Doc changes on 2.7

### CODE REVIEW

File: bundles/installation.rst [hunk 1]
```diff
@@ -108,14 +108,14 @@ via the ``config:dump-reference`` command:
 
 .. code-block:: bash
 
-    $ app/console config:dump-reference AsseticBundle
+    $ bin/console config:dump-reference AsseticBundle
```
Comment (xabbuh) [inline bundles/installation.rst]: This does only apply to Symfony >= 3.0.

File: bundles/override.rst [hunk 2]
```diff
@@ -105,17 +105,16 @@ associations. Learn more about this feature and its limitations in
 Forms
 -----
 
-In order to override a form type, it has to be registered as a service (meaning
-it is tagged as ``form.type``). You can then override it as you would override any
-service as explained in `Services & Configuration`_. This, of course, will only
-work if the type is referred to by its alias rather than being instantiated,
-e.g.::
+Form types are referred to by their fully-qualified class name::
```
Comment (xabbuh) [inline bundles/override.rst]: This is only true for Symfony >= 2.8.

Comment (xabbuh) [inline bundles/override.rst]: But we may want to remove this paragraph entirely for the 2.7 docs and only talk about form type extensions.

**Review by xabbuh — CHANGES_REQUESTED** [review]
@franciscocorrales I left some minor comments that should be addressed. But overall I like your suggestions.

**Comment by javiereguiluz** [comment]
@franciscocorrales thanks for contributing these improvements. The PR stalled and has conflicts, so I'm finishing it in #7490. I reused your original commits, so you'll get full credit for your work. Thanks!

**Pull request closed without merging**